proto/Makefile

```diff
@@ -23,6 +24,11 @@ proto: $(PROTO_GEN_SRC)
 	PROTOPATH=$(CURDIR) $(MAKE) -C service/fccontrol proto
 	PROTOPATH=$(CURDIR) $(MAKE) -C service/drivemount proto
 
+proto-docker:
+	docker run --rm \
```

Comment:
Does this result in the proto files being owned by root? If so, we could set `UID:=$(shell id -u)` in the makefile and then specify `--user $(UID)` for the docker run command (which we do for some of the other builders in this repo). Then users don't have to deal with files that require `sudo` to modify/delete

Reply:
It does! I think we need to add the GID also.